cmd/proxy/main.go

```diff
@@ -39,7 +41,7 @@ func parseBackupURL(backupUrl string) (string, string, error) {
 }
 
 func (ps *proxyServer) Download(ctx context.Context, req *pb.DownloadRequest) (*pb.DownloadResponse, error) {
-	setupLog.Info("Downloading file %q", req.BackupUrl)
+	ps.log.Info("Download", "url", req.BackupUrl)
```

Comment:
The proxy was panicking here because we weren't passing a key-value pair to the structured logging library:

```
kubectl -n eco-system logs eco-proxy-b85cd8cb-8ddxj  --follow
2020-02-26T13:39:28.577Z	INFO	setup	Starting etcd-cluster-controller upload/download Proxy service	{"version": "v0.2.0-10-g8fbfd62"}
2020-02-26T13:39:28.577Z	INFO	setup	Listening	{"grpc-address": ":8080"}



2020-02-26T13:40:51.696Z	DPANIC	setup	odd number of arguments passed as key-value pairs for logging	{"ignored key": "s3://backups.test.improbable.io/snapshot.db?endpoint=minio.minio.svc&disableSSL=true&s3ForcePathStyle=true"}

```